R/resolver.R

```diff
@@ -556,7 +556,22 @@ resolve_groups <- function(expr, vector) {
     return(":GRAND_SUMMARY:")
   }
 
+  # Handle groups = FALSE supplied to not do any summary rows.
+  if (isFALSE(resolved)) {
+    resolved <- NULL
+  }
+
   if (length(resolved) < 1) {
+    # 1292
+    input <- tryCatch(rlang::as_label(quo), error = NULL)
+    if (identical(input, "everything()")) {
+      cli::cli_warn(c(
+        "Since gt v0.9.0, the `groups = everything()` option has been deprecated in {.fn summary_rows}.",
+        "*" = "If this was intended for generation of grand summary rows, instead
+  use the `grand_summary_rows()` function."
+      ))
+      return(":GRAND_SUMMARY:")
```

Comment:
```suggestion
      cli::cli_abort(c(
        "Since gt v0.9.0, the `groups = everything()` option has been deprecated in {.fn summary_rows}.",
        "*" = "If this was intended for generation of grand summary rows, instead
  use the `grand_summary_rows()` function."
      ))
```

Since this is already broken, it would not be a breaking change to do this. (For consistency, I think it may be preferable?

Reply:
I like it! I'll commit the suggestion.